src/Modules/Data/Commands.Data.Test/GetPowerBITableTests.cs

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Linq;
+
+namespace Microsoft.PowerBI.Commands.Data.Test
+{
+    [TestClass]
+    public class GetPowerBITableTests
+    {
+        public static CmdletInfo GetPowerBITableCmdletInfo { get; } = new CmdletInfo($"{GetPowerBITable.CmdletVerb}-{GetPowerBITable.CmdletName}", typeof(GetPowerBITable));
+        public static CmdletInfo GetPowerBIDatasetCmdletInfo { get; } = new CmdletInfo($"{GetPowerBIDataset.CmdletVerb}-{GetPowerBIDataset.CmdletName}", typeof(GetPowerBIDataset));
+
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void GetPowerBITable_DatasetIdSetName()
+        {
+            using (var ps = System.Management.Automation.PowerShell.Create())
+            {
+                // Arrange
+                ProfileTestUtilities.ConnectToPowerBI(ps, "Public");
+                ps.AddCommand(GetPowerBIDatasetCmdletInfo).AddParameter("Scope", "Individual");
```

Comment:
Wherever possible, try to use the relevant enum instead of the string.
e.g.
Instead `"Scope", "Individual"` use `nameof(GetPowerBIDatasetCmdletInfo.Scope), PowerBIUserScope.Individual`
Instead of `"Public"` use `PowerBIEnvironmentType.Public`

For the last change, you may need to update ProfileTestUtilities.ConnectToPowerBI to take the PowerBIEnvironmentType class instead.

Reply:
I updated to use PowerBIUserScope.Individual for Scope and removed Public as its for my own test. Do we also need UnitTesting of DI version?

Reply:
I actually think it's better for us to use Public since it will enable contributors outside of Microsoft to run the unit tests more easily. Unit tests is solely for the DI constructor that uses the init factory.

Reply:
I changed it to use Public